dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -330,6 +333,53 @@ def kurtosis(self):
                     self.match_count,
                     self._biased_kurtosis)
 
+    @staticmethod
+    def _perform_t_test(mean1, var1, n1,
+                        mean2, var2, n2):
+        if n1 <= 1 or n2 <= 1:
+            warnings.warn("Insufficient sample size to "
+                          "perform t-test.", RuntimeWarning)
+            return "null"
```

Comment:
I think maybe just return `None` rather than text. JSON treats `None` as `null` during serialization.

Reply:
or rather we can could also consider putting the if below the `results` and just return that such that static return in terms of keys